addon/components/bs-modal/dialog.hbs

```diff
@@ -1,28 +1,40 @@
 <div
-  role="dialog"
-  tabindex="-1"
+  role='dialog'
```

Comment:
Need to revert the quote style changes in this file. I guess this happened by running Prettier on the file but not having Prettier configured for Handlebars in the repository.